packages/block-editor/src/components/block-preview/index.js

```diff
@@ -31,18 +32,37 @@ function BlockPreview( props ) {
 	);
 }
 
-export function BlockPreviewContent( { name, attributes } ) {
-	const block = createBlock( name, attributes );
-	return (
-		<Disabled className="editor-block-preview__content block-editor-block-preview__content editor-styles-wrapper" aria-hidden>
-			<BlockEdit
-				name={ name }
-				focus={ false }
-				attributes={ block.attributes }
-				setAttributes={ noop }
-			/>
-		</Disabled>
-	);
+export class BlockPreviewContent extends Component {
```

Comment:
This file contains 2 components. Wouldn't it better to handle this error in the parent `BlockPreview` component and don't review this part of UI at all?

Aside: it might be nice to split this into two files.

Reply:
Actually, I think it's better to handle as closely as possible to the error. That way the "title" of the preview is still shown properly and just the content is left empty.